### PR TITLE
fix: remove policy panic

### PR DESCRIPTION
**Pull request by caoxianfeng**

### CODE REVIEW

File: model/policy.go [hunk 1]
```diff
@@ -247,7 +247,12 @@ func (model Model) RemovePolicy(sec string, ptype string, rule []string) bool {
 		return false
 	}
 
-	model[sec][ptype].Policy = append(model[sec][ptype].Policy[:index], model[sec][ptype].Policy[index+1:]...)
+	if index < len(model[sec][ptype].Policy)-1 {
```
Comment (hsluoyz) [inline model/policy.go:250]: What does this code mean?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@caoxianfeng

File: model/policy.go (hunk 1 above)
Comment (caoxianfeng) [reply model/policy.go:250]: Ensure that the index exists in the slice, and using copy is more efficient than append and can reduce memory allocation.

Comment (caoxianfeng) [reply model/policy.go:250]: @hsluoyz

Comment (caoxianfeng) [reply model/policy.go:250]: image

Comment (hsluoyz) [reply model/policy.go:250]: Use append instead of copy. Readability is most important

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@MuZhou233 plz review

**Pull request closed without merging**